pysam/libcalignmentfile.pyx

```diff
@@ -127,7 +128,8 @@ KNOWN_HEADER_FIELDS = {"HD" : {"VN" : str, "SO" : str, "GO" : str},
 # the end as parsing a CL will ignore any subsequent records.
 VALID_HEADER_ORDER = {"HD" : ("VN", "SO", "GO"),
                       "SQ" : ("SN", "LN", "AS", "M5",
-                               "UR", "SP", "AH"),
+                              "UR", "SP", "AH", "TP",
+			      "DS", "AN"),
```

Comment:
These are not in the same order as the spec, so we could take the time to update them?
![Screenshot 2023-10-18 at 4 53 45 PM](https://github.com/pysam-developers/pysam/assets/846274/1e7411ae-504d-4648-85dd-16f14f3ffd65)